lib/jazzy/source_declaration/type.rb

```diff
@@ -61,6 +61,14 @@ def extension?
         kind =~ /^source\.lang\.swift\.decl\.extension.*/
       end
 
+      def extensible?
```

Comment:
Updating with `swift_` and `objc_` prefixes as you describe for this PR.

What do you think about generic tests like `protocol?` and `extensible?` plus orthogonal language tests like `objc?` and `swift?`?